simtools/utils/general.py

```diff
@@ -165,7 +165,7 @@ def collect_data_from_file_or_dict(file_name, in_dict, allow_empty=False):
         return dict(in_dict)
 
     if allow_empty:
-        _logger.debug("Input has not been provided (neither by file, nor by dict)")
+        _logger.warning("Input has not been provided (neither by file, nor by dict)")
```

Comment:
Keep this at debug: we explicitly `allow_empty`, so there is no need to warn that it actually happens.